doc/api/deprecations.md

```diff
@@ -3774,6 +3774,19 @@ Type: Runtime
 
 The `util.getCallSite` API has been removed. Please use [`util.getCallSites()`][] instead.
 
+### DEP0187: `fs.existsSync` never throw on invalid arguments
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55753
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
```

Comment:
We should first land it as doc-only

Reply:
Should I open a different PR for that?

Reply:
You can, we'll need two PRs anyway, you can either keep this one for the runtime and open a new one for the doc-only, or convert this one to be doc-only and open another one for the runtime deprecation later.